docs/manual/working/scalaGuide/main/json/ScalaJsonAutomated.md

```diff
@@ -9,6 +9,14 @@ The following macro will create a `Reads[Resident]` based on its structure and t
 
 @[auto-reads](code/ScalaJsonAutomatedSpec.scala)
 
+The [value classes](https://docs.scala-lang.org/overviews/core/value-classes.html) are also supported. Given the following value class, based on a `String` value:
+
+@[valueClass](code/ScalaJsonAutomatedSpec.scala)
+
+Then it's also possible to generate a `Reads[IdText]` using the following macro (as `String` is already supported):
+
+@[value-reads](code/ScalaJsonAutomatedSpec.scala)
+
 When compiling, the macro will inspect the given class and
 inject the following code, exactly as if you had written it manually:
```

Comment:
The way this is written now it looks like this is could be about value classes. I'd move the value class stuff to a different section, and explicitly say that a value class will be serialized exactly as the wrapped type is serialized.